### PR TITLE
Release v3.7.0

### PR DESCRIPTION
**Pull request by neiljerram**

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 40a4ceada0e64c739fa5b52aea011eff561c9e3d

https://deploy-preview-2584--calico.netlify.com

File: _includes/v3.7/release-notes/v3.7.0-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+While IP in IP is slightly more efficient in terms of encapsulation
+overhead, there are some networks that do not allow IP in IP traffic,
+for example Azure and DigitalOcean. VXLAN support in Calico means you
+can now use Calico to network workloads in these environments.
+
+ - Felix programs VXLAN routes [felix #1989](https://github.com/projectcalico/felix/pull/1989) (@caseydavenport)
+ - VXLAN encapsulation can now be configured per IP pool [libcalico-go #1065](https://github.com/projectcalico/libcalico-go/pull/1065) (@caseydavenport)
+
+For more information, see the Calico documentation.
```
Comment (caseydavenport) [inline _includes/v3.7/release-notes/v3.7.0-release-notes.md]: Might be worth including a link to the VXLAN doc here.

File: _includes/v3.7/release-notes/v3.7.0-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#### Optimized denial-of-service protection for host endpoints using XDP
+
+Calico now automatically enforces specific types of blacklist policies
+at the earliest possible point in the packet processing pipeline,
+including offloading to NIC hardware whenever possible.
+
+ - Dataplane: optimize early host-endpoint blacklist rules [felix #1991](https://github.com/projectcalico/felix/pull/1991) (@iaguis)
+ - [calico #2546](https://github.com/projectcalico/calico/pull/2546) (@iaguis)
+
+For more information, see the HOWTO guide and our upcoming blog post.
```
Comment (caseydavenport) [inline _includes/v3.7/release-notes/v3.7.0-release-notes.md]: Similarly here - could link to https://docs.projectcalico.org/v3.7/security/defend-dos-attack

File: _includes/v3.7/release-notes/v3.7.0-release-notes.md (hunk 1 above)
Comment (lwr20) [reply _includes/v3.7/release-notes/v3.7.0-release-notes.md]: i.e. here: https://docs.projectcalico.org/v3.7/networking/vxlan-ipip

File: _includes/v3.7/release-notes/v3.7.0-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Calico v3.7.0 introduces support for the NetworkSet resource, which
+allows for the same functionality but within the scope of a particular
+Namespace. This is useful if you want to be able to reference a set of
+IP addresses using a selector from a namespaced network policy
+resource.
+
+- Add NetworkSet resource. NetworkSets are the namespaced-equivalent of GlobalNetworkSet. [libcalico-go #1055](https://github.com/projectcalico/libcalico-go/pull/1055) (@lmm)
+
+For more information, see the Calico documentation.
```
Comment (lwr20) [inline _includes/v3.7/release-notes/v3.7.0-release-notes.md]: Maybe link to: https://docs.projectcalico.org/v3.7/reference/calicoctl/resources/networkset

**Pull request merged**